recipes/fates-emerald/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "fates-emerald" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/NordicESMhub/ctsm/archive/release-emerald-platform{{ version }}.tar.gz
+    sha256: 380c7c46377a31aa6c32b9b2e3313a9ff03d3f86caa164dc369a3561e3cd9686
+
+build:
+    skip: True  # [win or osx]
+    number: 0
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - cmake
+    host:
+        - python
+        - mpich
+        - hdf5
+        - netcdf-fortran
+        - svn
```

Comment:
is svn really needed?

Reply:
I just tested without and it works. So I removed it. Thanks.